docker-compose.yml

```diff
@@ -138,13 +141,20 @@ services:
       spnet:
 
   influxdb:
-    image: fogsyio/influxdb:1.7
+    image: influxdb:2.6
     environment:
       - INFLUXDB_DATA_ENGINE=tsm1
       - INFLUXDB_REPORTING_DISABLED=false
       - INFLUXDB_ADMIN_ENABLED=true
+      - DOCKER_INFLUXDB_INIT_USERNAME=admin
```

Comment:
Do we want to align the naming style here with the envs of couchdb?

Reply:
These are the env variables that are supported by the official image, so we can't change these unfortunately

Reply:
Makes sense 💡